package.json

```diff
@@ -15,7 +15,7 @@
     "react-collapse": "^5.1.0",
     "react-dom": "^17.0.2",
     "react-emoji-render": "^1.2.4",
-    "react-scripts": "4.0.3",
+    "react-scripts": "^5.0.1",
```

Comment:
any reasons for changing the version? please mention in the description of the PR